packages/documentation/src/stories/utilities/opacity/opacity.styles.scss

```diff
@@ -0,0 +1,13 @@
+@use '@swisspost/design-system-styles/core' as post;
+
+.opacity-example > div {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  width: 100px;
+  height: 100px;
+  margin: 2vw;
```

Comment:
I guess you added the margin for the snapshot page but it impacts the documentation page as well so it's better to remove it.
```suggestion
```